run_mypy.py

```diff
@@ -11,6 +11,7 @@
            "axelrod/moran.py",
            "axelrod/plot.py",
            "axelrod/random_.py",
+           "axelrod/result_set.py"
```

Comment:
You are missing a `,` here.

Information about running the type check tests is available here: http://axelrod.readthedocs.io/en/stable/tutorials/contributing/running_tests.html#type-checking